### DISCUSSION TITLE
using tauri sveltekit app with hono client

### DISCUSSION THREAD

**Discussion by islamzaoui** [tauri-apps/tauri#11714] [Q&A]
Hello

I have a monorepo that has 2 apps, a backend with hono & bunjs and a frontend with Tauri and sveltekit
I'm using the hono client to fetch from the backend but this causes a CORS error, is there any way to make only the Tauri app fetch from the server?

my current implementation:
```ts
const app = new Hono<App.Env>()
	.basePath('/api')
	.use(
		cors({
			origin: '*', // using  "*" is my issue
			allowMethods: ['GET', 'POST', 'PUT', 'DELETE'],
			allowHeaders: ['Content-Type']
		})
	)
```

**Comment by oghaAI** [comment]
Did you figure this out ?
